src/NzbDrone.Core/Music/RefreshAlbumService.cs

```diff
@@ -67,11 +67,12 @@ public void RefreshAlbumInfo(Artist artist, IEnumerable<Album> remoteAlbums)
                     albumToUpdate.CleanTitle = album.CleanTitle;
                     //albumToUpdate.TrackNumber = album.TrackNumber;
                     albumToUpdate.Title = album.Title ?? "Unknown";
+                    albumToUpdate.CleanTitle = Parser.Parser.CleanArtistTitle(albumToUpdate.Title);
                     //albumToUpdate.AlbumId = album.AlbumId;
                     //albumToUpdate.Album = album.Album;
                     //albumToUpdate.Explicit = album.Explicit;
                     albumToUpdate.ArtistId = artist.Id;
-                    albumToUpdate.Path = artist.Path + album.Title;
+                    albumToUpdate.Path = artist.Path + "\\" + album.Title;
```

Comment:
Should be able to use getalbumfolder in file name builder to get correct album path based on naming settings.